api_tests/caching/test_caching.py

```diff
@@ -0,0 +1,239 @@
+import copy
+import json
+from django.conf import settings
+
+from datadiff import diff
+from requests.auth import HTTPBasicAuth
+
+from tests.base import ApiTestCase
+import requests
+
+class TestVarnish(object):
+
+    local_varnish_base_url = 'http://localhost:8193/v2/'
+    local_python_base_url = 'http://localhost:8000/v2/'
+
+    def setUp(self):
+        if not settings.RUN_VARNISH_IN_DEV:
+            return
+        self.authorization = HTTPBasicAuth(u'mocha@osf.io', u'password')
+
+    def test_compare_python_responses_to_varnish_responses(self):
+        if not settings.RUN_VARNISH_IN_DEV:
+            return
+        querystrings = dict(
+            nodes=[
+                u'comments',
```

Comment:
`from __future__ import unicode_literals` will make all strings in the file unicode by default.